circle.yml

```diff
@@ -0,0 +1,23 @@
+machine:
+  timezone: UTC
+  java:
+    version: oraclejdk8
+  environment:
+    _JAVA_OPTIONS: "-Xms512m -Xmx1024m"
+dependencies:
+  override:
+    - mvn dependency:go-offline --fail-never
```

Comment:
Hm, looks like `mvn dependency:go-offline` still fails on `Dropwizard End-to-end Tests` 
https://circleci.com/api/v1.1/project/github/dropwizard/dropwizard/25/output/11/0?file=true